packetbeat/config/config.go

```diff
@@ -59,64 +65,72 @@ type Icmp struct {
 	TransactionTimeout *int  `yaml:"transaction_timeout"`
 }
 
-type Dns struct {
-	ProtocolCommon      `yaml:",inline"`
-	Include_authorities *bool
-	Include_additionals *bool
+// DNS holds DNS specific configuration information
+type DNS struct {
+	ProtocolCommon     `yaml:",inline"`
+	IncludeAuthorities *bool
+	IncludeAdditionals *bool
 }
 
-type Http struct {
-	ProtocolCommon       `yaml:",inline"`
-	Send_all_headers     *bool
-	Send_headers         []string
-	Split_cookie         *bool
-	Real_ip_header       *string
-	Include_body_for     []string
-	Hide_keywords        []string
-	Redact_authorization *bool
+// HTTP holds HTTP specific configuration informaiton
+type HTTP struct {
+	ProtocolCommon      `yaml:",inline"`
+	SendAllHeaders      *bool
+	SendHeaders         []string
+	SplitCookie         *bool
+	RealIPHeader        *string
+	IncludeBodyFor      []string
+	HideKeywords        []string
+	RedactAuthorization *bool
 }
 
+// Memcache holds Memcached specific configuration informaiton
 type Memcache struct {
 	ProtocolCommon        `yaml:",inline"`
 	MaxValues             int
 	MaxBytesPerValue      int
-	UdpTransactionTimeout *int
+	UDPTransactionTimeout *int
```

Comment:
Previous comment applies to all config changes. Best is to add the yaml part to every single entry, so there will not be any problems in the future.